inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/Constants.java

```diff
@@ -73,11 +79,48 @@ public final class Constants {
      * Node id used in inlong metric
      */
     public static final String NODE_ID = "nodeId";
+    /**
+     * Database Name used in inlong metric
+     */
+    public static final String DATABASE_NAME = "database";
+    /**
+     * Table Name used in inlong metric
+     */
+    public static final String TABLE_NAME = "table";
+    /**
+     * Schema Name used in inlong metric
+     */
+    public static final String SCHEMA_NAME = "schema";
+    /**
+     * Topic Name used in inlong metric
+     */
+    public static final String TOPIC_NAME = "topic";
+
     /**
      * It is used for inlong.metric
      */
     public static final String DELIMITER = "&";
 
+    /**
+     * It is used for metric data to build schema identify
+     */
+    public static final String SEMICOLON = ".";
+
+    /**
+     * It is used for metric data to spilt schema identify
+     */
+    public static final String SPILT_SEMICOLON = "\\.";
+
+    /**
+     * This indicator represents the snapshot phase during the read phase
+     */
+    public static final Long SNAPSHOT_PHASE = 1L;
```

Comment:
suggest use enum for snapshot or incremental

Reply:
> suggest use enum for snapshot or incremental

good idea ,i will do it .